course_discovery/settings/base.py

```diff
@@ -635,3 +635,5 @@
     'MEDIA_ROOT': MEDIA_ROOT,
     'MEDIA_URL': MEDIA_URL
 }
+WORDPRESS_USERNAME = 'discovery_service_worker'
+WORDPRESS_APP_PASSWORD = ''
```

Comment:
From where we will get the password? Also will it be common for all sites?

Reply:
@zubair-arbi we create a user and application in the WordPress. Yes, the password and username will remain for all the sites. The only important point that we have to give access of all sites to that service worker. 

I will create a detailed document for this.